### ISSUE TITLE
[Bug]: Accessibility - add a header to the large title.

### ISSUE THREAD

**Issue by Tayebsed93** [Orange-OpenSource/ods-flutter#343]
### Prerequisites

- [X] I have [searched the backlog](https://github.com/Orange-OpenSource/ods-flutter/issues) for duplicate or closed feature requests

### Your test device

iPhone 14 pro Max

### OS Version

iOS 17.0

### App version

v0.8.0

### Describe the issue

The title is not being vocalized as a header with VoiceOver

### Expected Behavior

The title should be vocalized with the screen reader
